scavenger-agent-java/src/integrationTest/java/integrationTest/util/AgentLogAssertionUtil.java

```diff
@@ -0,0 +1,31 @@
+package integrationTest.util;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.matchesPattern;
+
+import java.util.regex.Pattern;
+
+public class AgentLogAssertionUtil {
+    public static void assertSampleAppOutput(String stdout) {
```

Comment:
assertSampleAppOutput is only used for ConfigTest
I believe hiding asserting logic in different files is not good idea. 
It makes the validation not explicit.

Reply:
`ScanTest` and `InvocationTest` also uses `assertSampleAppOutput()`, so I've found it to difficult to just inline this method to the call sites.